change/@fluentui-react-spinbutton-803b9a0b-82dd-471f-b57d-7717eb0d925d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Merge branch 'master' of https://github.com/microsoft/fluentui into spin-button",
```

This comment will be part of the public changelog so it needs to be something that make sense in that context. What do you think about?
```suggestion
  "comment": "docs: add accessibility documentation for SpinButton",
```